js_modules/dagit/packages/ui/src/components/StyledButton.tsx

```diff
@@ -4,17 +4,34 @@ import {IconWrapper} from './Icon';
 import {SpinnerWrapper} from './Spinner';
 import {FontFamily} from './styles';
 
+export type ButtonJoinState = 'left' | 'right' | 'both';
```

Comment:
Slightly unclear what "join state" means without investigating the code, or using trial-and-error. Does `'left'` mean that the button is joined on the left of the button set, or that the "joined" side is on the left side of the button? In the Storybook example, it's a little counterintuitive that the "left" button has `joined="right"`, etc.

It makes me think it might be clearer being called `flatSide: 'left' | 'both' | 'right'` or something really dead obvious like that. What do you think?

Reply:
Tentatively changed to `joinedSide` - I think that makes it clearer what the prop does, but maybe it's still not clear enough?